public/home.html

```diff
@@ -70,4 +84,9 @@ <h3 class="text-center">Contracts</h3>
     tableStakers.updateOrAddData(GGP.stakersAsTabulatorData());
   });
 </script>
+<style>
```

Comment:
🔪
prefer tailwind classes over style

Reply:
I don't want to add tailwind to this project........ yet. If it gets much more complicated I could get behind that. But for now we're trying to keep things small and vanilla.

Reply:
Yeah I don't think you should. i saw the weird class names and assumed you already had it installed.